apps/webpack.js

```diff
@@ -20,7 +21,9 @@ var baseConfig = {
     "blockly": "this Blockly",
     "react": "var React",
     "react-dom": "var ReactDOM",
-    "jquery": "var $"
+    "jquery": "var $",
+    "radium": "var Radium",
+    "bindings": true
```

Comment:
I guess I'm surprised this works at all - isn't the code that used to put these on `window` now affected by this global substitution?  How is it getting the thing to create the global?